.github/workflows/docs.yml

```
@@ -28,16 +28,23 @@ jobs:
       - name: Get yarn cache directory path
         id: yarn-cache-dir-path
         run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/cache@v1
+      - uses: actions/cache@v2
         with:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('docs/yarn.lock') }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
       - run: yarn install --frozen-lockfile
       - run: yarn danger ci
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/cache@v2
+        with:
+          path: .next
+          key: next-${{ hashFiles('yarn.lock') }}-${{ hashFiles('pages/**') }}
+          restore-keys: |
+            next-${{ hashFiles('yarn.lock') }}
+            next-
```

Small suggestion: I think it's worth qualifying the cache key with "docs" just so that if we were to add another Next.js site & cache to this repo, the cache keys wouldn't collide.

```suggestion
      - uses: actions/cache@v2
        with:
          path: .next
          key: docs-next-${{ hashFiles('yarn.lock') }}-${{ hashFiles('pages/**') }}
          restore-keys: |
            docs-next-${{ hashFiles('yarn.lock') }}
            docs-next-
```

---

Good idea.